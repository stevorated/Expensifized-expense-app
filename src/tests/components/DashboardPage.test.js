import React from 'react';
import { shallow } from 'enzyme';
import {createSerializer} from 'enzyme-to-json';
import toJson from 'enzyme-to-json';
import DashboardPage from '../../components/DashboardPage'

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

test('should render DashboardPage Component correctly', ()=>{
    const wrapper = shallow(<DashboardPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
})
 