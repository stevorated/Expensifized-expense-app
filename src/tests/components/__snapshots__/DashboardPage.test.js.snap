// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardPage Component correctly 1`] = `
<div>
  <ConnectFunction />
  <ConnectFunction />
  <ConnectFunction />
</div>
`;
